src/vmspawn/vmspawn.c

```diff
@@ -1411,6 +1446,43 @@ static int run_virtual_machine(int kvm_device_fd, int vhost_device_fd) {
                         return log_oom();
         }
 
+        if (arg_forward_journal) {
+                _cleanup_free_ char *sd_journal_remote = NULL, *listen_address = NULL, *cred = NULL;
+                r = find_executable("systemd-journal-remote", &sd_journal_remote);
```

Comment:
It doesn't seem to work. `systemd-journal-remote` is installed to `/usr/lib/systemd` and `find_executable` can't find it there. I got past this by hardcoding the path.

Reply:
I'm guessing it's tied to mkosi where `/usr/lib/systemd` is added to `PATH`. Either way `udevadm` segfaulted in one of machines I ran with `vmspawn` manually and I haven't been able to extract the journal/coredumps from there.

Reply:
-> #32739.